lua/telekasten.lua

```diff
@@ -1082,7 +1141,7 @@ end
 -- find today's daily note and create it if necessary.
 --
 local function GotoToday(opts)
-    opts = opts or calenderinfo_today()
+    opts = opts or calculate_dates()
```

Comment:
... and this also seems correct .... there is one omission which I'll write in the overall review text

Reply:
Let me keep reading through this, but so far I'm not seeing in.  In this case all I did was replace calendarinfo_today with calculate_dates, but I didn't change the signature.  

I do get now how if opts is not nil that the dates won't be calculated.  I think I was getting confused because `calendarinfo_today` had a local table named `opts` that I thought was your "outputs".  I didn't realize you're passing options around. 🤣